docusaurus/docs/guides/deployments/10-linux/10-controller/15-migrate.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Migrate a Controller Installation
+sidebar_label: Migrate
+---
+
+You can generate a configuration with the `ziti create config controller` command, optionally mutating the result through command-line options or environment variables. You can also find an annotated sample config file from [the Ziti repo](https://github.com/openziti/ziti/blob/main/etc/ctrl.with.edge.yml).
```

Comment:
This section should just talk about "if you already have a configuration you like and you want to bring it in alignment with the new packaging, here's how you do it" vibe and not talk about `ziti create config` or any ofthat. 

If you like the answer file approach, it seems like a migration could be part of that answer file... You specify the things needed and then on startup the process 'migrates' the configuration on your behalf? (but doesn't delete the existing configuration)

That or this should just be a set of tasks, enumerated, with "for example" type commands that show them one way to accomplish the task.

Reply:
I like the idea of fragmenting this BASH script into a more instructive set of steps.

Migrating a complete configuration and the bootstrapping answer file are mutually-exclusive. Migrating a partial configuration (e.g., migrating only the db, but generating the rest) would involve answering the bootstrapping questions pertinent to the parts of the configuration that are to be generated (e.g., the PKI and config.yml), and any answers pertaining to the migrated db (i.e., `ZITI_PWD`) would be ignored during bootstrapping.